dandiapi/api/models/zarr.py

```diff
@@ -22,7 +23,9 @@
 
 
 class ZarrUploadFileManager(models.Manager):
-    def create_zarr_upload_file(self, zarr_archive: ZarrArchive, path: str, **kwargs):
+    def create_zarr_upload_file(
+        self, zarr_archive: ZarrArchive | EmbargoedZarrArchive, path: str, **kwargs
```

Comment:
TIL you could use the | operator for syntax :exploding_head:

Reply:
Unfortunately, it seems like this syntax is only valid in Python 3.10 - https://www.python.org/dev/peps/pep-0604/ (dandi-api is on 3.8).

Reply:
I think `from __future__ import annotations` will enable it, and that is conveniently already included in this file. Any objections to using it here?